FindChannelFunction/build.gradle

```
@@ -5,15 +5,19 @@ plugins {
     id 'pmd'
 }
 
+sourceCompatibility =  JavaVersion.VERSION_11  // source-code version and must be <= targetCompatibility
+targetCompatibility = JavaVersion.VERSION_11 // bytecode target version
+
 repositories {
     jcenter()
 }
 
 
+
 dependencies {
     implementation 'com.amazonaws:aws-lambda-java-core:1.2.0'
-    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.10.1'
-    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: '2.10.1'
+    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.10.3'
```

create a variable and use the variable 
```suggestion
def jacksonVersion = "2.10.3"

dependencies{
implementation "com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}"
...
}
        implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.10.3'
```

---

But use group: ... formatting please.

---

Reformatted